cumulusci/cumulusci.yml

```diff
@@ -344,6 +344,10 @@ tasks:
         description: Apply org settings from a scratch org definition file or dict
         class_path: cumulusci.tasks.salesforce.org_settings.DeployOrgSettings
         group: Salesforce DX
+    promote_2gp_package:
```

Comment:
I'd suggest calling this promote_package_version for congruence with create_package_version. (And change the Python class to match.) Not explicitly saying 2gp is intentional since 2gp is meant to be the primary packaging method in the future.